core/node/api_server/src/web3/namespaces/eth.rs

```diff
@@ -385,6 +385,8 @@ impl EthNamespace {
         address: Address,
         block_id: Option<BlockId>,
     ) -> Result<Bytes, Web3Error> {
+        const EVM_BYTECODE_MARKER: u8 = 2;
```

Comment:
nit: we have this const in two places. Would it be better to import it from `core/lib/utils/src/bytecode.rs`?